src/dispatchers/pageDispatcher.ts

```diff
@@ -38,13 +38,21 @@ import { Download } from '../server/download';
 export class PageDispatcher extends Dispatcher<Page, channels.PageInitializer> implements channels.PageChannel {
   private _page: Page;
 
+  private static from(scope: DispatcherScope, page: Page | undefined): PageDispatcher | undefined {
```

Comment:
We called it createNullable in ElementHandle

Reply:
done